stingray/deadtime/model.py

```diff
@@ -179,6 +304,9 @@ def pds_model_zhang(N, rate, td, tb, limit_k=60):
         Limit to this value the number of terms in the inner loops of
```

Comment:
I think this explanation is crucial, maybe you can write a very short example in the docstring on how to call check_A and check_B